src/app.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function coppyFile(fileToCopy, newFile) {
+  if (fileToCopy === newFile) {
+    return;
+  }
+
+  const dataToWrite = fs.readFileSync(fileToCopy, { encoding: 'utf-8' });
```

Comment:
Not all files are utf-8. Check the docs, you can omit that parameter to read raw bytes
```suggestion
  const dataToWrite = fs.readFileSync(fileToCopy);
```